packages/app-project/src/shared/components/ContentBox/ContentBox.js

```diff
@@ -53,13 +53,11 @@ function ContentBox (props) {
           )}
 
           {(linkLabel && linkProps) && (
-            <Link {...linkProps} passHref>
-              <Anchor>
-                <SpacedText>
-                  {linkLabel}
-                </SpacedText>
-              </Anchor>
-            </Link>
+            <Anchor {...linkProps}>
+              <SpacedText>
+                {linkLabel}
```

Comment:
Not related to this bug fix, but would it make more sense if the link text was part of the link props?
```js
linkProps = {
  href: '/projects/test/owner/stats',
  label: 'View more stats'
}
```
I'm guessing it's passed separately because `linkProps` was meant to represent a set of props for [`next/link`](https://nextjs.org/docs/api-reference/next/link), which are different from the props for Grommet's [Anchor](https://v2.grommet.io/anchor).

Reply:
True. I think `NavLink` does that, but the check will break with the new router API. #1973